HIP/hip-412.md

```diff
@@ -231,7 +393,7 @@ This is an example of a basic metadata as described by this schema. Format "none
     }
 }
 ```
-#### Example Schema: none, video
+### Example Schema: none, video
```

Comment:
Need to make sure that all the examples follow the spec. They should have an `image` in the root and the `type`.